_sass/base/extends/_centered.scss

```diff
@@ -0,0 +1,5 @@
+// this is kind of silly, but ¯\_(ツ)_/¯
```

Comment:
Yep. But it's OK. Now we can use it....EVERYWHERES and it's gonna be one nice, neat chunk o' CSS when it compiles.